helm/templates/_helpers.tpl

```diff
@@ -5,6 +5,66 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Name of k8s cluster
+*/}}
+{{- define "kubernetes-cluster.name" -}}
+{{- if empty .Values.clusterName }}
+{{- default "" (printf "k8s-cluster-%s" (sha256sum .Release.Name | trunc 7)) }}
+{{- else }}
+{{- default "" .Values.clusterName }}
+{{- end }}
+{{- end }}
+
+{{/*
+Helper function to modify cloudwatch-agent config
```

Comment:
Whats diff between the two funcs below? 
I would have expected a generic helper function that takes a config (either user provided or default) as an input and then just merges region & cluster name on top of that.
```
.Values.agent.config | injectRegionAndClusterName | toJson | quote
.Values.agent.defaultConfig | injectRegionAndClusterName | toJson | quote
```